app/jobs/create_gemini_recipient_ids_job.rb

```diff
@@ -0,0 +1,31 @@
+# Creates the Uphold Cards for a publisher
+class CreateGeminiRecipientIdsJob < ApplicationJob
+  queue_as :default
+
+  def perform(gemini_connection_id:)
+    gemini_connection = GeminiConnection.find(gemini_connection_id)
+    # Users aren't able to create a recipient id if they are not fully verified
+    if gemini_connection.payable?
+      recipient = Gemini::RecipientId.find_or_create(token: gemini_connection.access_token)
+      gemini_connection.update(recipient_id: recipient.recipient_id)
+      gemini_connection.update_default_currency
+
+      gemini_connection.publisher.channels.each do |channel|
+        gemini_connection_for_channel = GeminiConnectionForChannel.where(
+          gemini_connection: gemini_connection,
+          currency: gemini_connection.default_currency,
+          channel_identifier: channel.details.channel_identifier
+        ).first_or_create(channel_id: channel.id)
+
+        channel_recipient = Gemini::RecipientId.find_or_create(token: gemini_connection.access_token, label: channel.id)
```

Comment:
```suggestion
        channel_recipient = Gemini::RecipientId.find_or_create(token: gemini_connection.access_token, label: channel.details.brave_publisher_id)
```
My impression is that the label could be created on Gemini's server, similar to Uphold card's name.

Reply:
That's fine as an identifier.